search/Property.cc

```diff
@@ -1163,10 +1180,18 @@ getProperty(Clock *clk,
     return PropertyValue(clk->period(), sta->units()->timeUnit());
   else if (stringEqual(property, "sources"))
     return PropertyValue(clk->pins());
+  else if (stringEqual(property, "generated"))
+    return PropertyValue(clk->isGenerated());
+  else if (stringEqual(property, "virtual"))
+    return PropertyValue(clk->isVirtual());
   else if (stringEqual(property, "propagated"))
```

Comment:
redundant with is_propagated

Reply:
STA tools tend to prefer `is_X` prefix, is it ok to use this convention and eliminate the `X` style properties?

Reply:
yes